integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -104,7 +107,20 @@ object PluginImplementation {
       val sourceDirs = Keys.sourceDirectories.value
       val testFrameworks = Keys.testFrameworks.value.map(_.implClassNames)
       // TODO(jvican): Override classes directories here too (e.g. plugins are defined in the build)
-      val scalacOptions = Keys.scalacOptions.value
+
+      val scalacOptions = {
+        val scalacOptions0 = Keys.scalacOptions.value
+        val internalClasspath = AutoImported.bloopInternalClasspath.value
+        internalClasspath.foldLeft(scalacOptions0) {
+          case (scalacOptions, (oldClassesDir, newClassesDir)) =>
+            scalacOptions.map { scalacOption =>
+              scalacOption
+                .replace(oldClassesDir.toString, newClassesDir.getAbsolutePath)
```

Comment:
I imagine that it doesn't matter, but is it on purpose that you pass the absolute path here?

Reply:
Yes.